build/scripts/tests.sh

```diff
@@ -69,7 +69,8 @@ do
     echo Running "${runtime} ${file_name[@]}"
     if [[ "${runtime}" == "dotnet" ]]; then
         runner="dotnet exec --depsfile ${deps_json} --runtimeconfig ${runtimeconfig_json}"
-        if [[ "${file_name[@]}" == *'Roslyn.Compilers.CSharp.Emit.UnitTests.dll' ]] || [[ "${file_name[@]}" == *'Roslyn.Compilers.VisualBasic.Symbol.UnitTests.dll' ]]
+        if [[ "${file_name[@]}" == *'Roslyn.Compilers.CSharp.Emit.UnitTests.dll' ]] || [[ "${file_name[@]}" == *'Roslyn.Compilers.VisualBasic.Symbol.UnitTests.dll' ]] || [[ "${file_name[@]}" == *'Roslyn.Compilers.CSharp.Semantic.UnitTests.dll' ]]
```

Comment:
Nit: consider breaking up long line. It looks fine on my 4k monitor, but would not on another monitor.

Reply:
Do you know what the newline rules are for bash script? I tried looking it up but couldn't find anything. If I knew the rule I'd break it up accordingly

Reply:
iirc either backslashes or evaluate conditions ahead of time into a variable and check the variable

---
In reply to: [196973329](https://github.com/dotnet/roslyn/pull/28030#discussion_r196973329) [](ancestors = 196973329)

Reply:
Tried backslash and it ended up giving syntax errors. Will address this in a future PR when I get back to making these all run on Linux and I have bash readily available to rapidly try this out.